x-pack/plugins/siem/public/components/navigation/index.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+
+import { setBreadcrumbs } from './breadcrumbs';
+import { TabNavigation } from './tab_navigation';
+
+export class SiemNavigationComponent extends React.Component<RouteComponentProps> {
```

Comment:
Nice! Thanks for cleaning up and consolidating this piece of navigation -- much easier to follow! :)